test/csuite/wt10897_compact_quick_interrupt/main.c

```diff
@@ -49,24 +49,6 @@ static bool interrupted_compaction = false;
 static bool skipped_compaction = false;
 static bool do_interrupt_compaction = false;
 
-/*
- * error_handler --
- *     Error handler.
- */
-static int
-error_handler(WT_EVENT_HANDLER *handler, WT_SESSION *session, int error, const char *message)
```

Comment:
I'd personally still keep the error handler (and just remove the error condition), so that the test can notice when there is a genuine error. I'm not sure it's necessary though—as long as errors during the test will be recorded somewhere properly.

Reply:
> I'm not sure it's necessary though

I am not sure either, see [here](https://github.com/wiredtiger/wiredtiger/blob/develop/src/support/err.c#L153) where we set a default error handler. If the test goes wrong, a message will be printed out to help with the diagnosis.